scripts/overlap.py

```diff
@@ -1,9 +1,11 @@
 """Computes the overlap of molecules between two datasets."""
 
 from argparse import ArgumentParser, Namespace
+from os.path import dirname, realpath
```

Comment:
This is building off the comment below.

```suggestion
import os
```